puppet/modules/slave/files/pbuilder_F60addforemanrepo

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+# Add the right theforeman.org repositories
+
+set -eu
+
+if [ -z "${REPO_SETUP:-}" ]; then
```

Comment:
Should we make this fail hard if it isn't set? In our builds we basically always want it anyway. Perhaps it should be 
```suggestion
if [ "${REPO_SETUP:-}" = 0]; then
```

Reply:
My idea was: don't fail hard *now*, allowing us to deploy this and still play around with the jobs and when we feel comfortable with the actual flow, just drop the `if` altogether.

Reply:
Can we really? The filename is the same as the one in the current jobs so it'll flip back and forth between a job and what Puppet applies.

Reply:
implementation details ;)